quarkus/runtime/src/main/java/org/keycloak/provider/quarkus/KeycloakPropertiesConfigSource.java

```diff
@@ -125,6 +125,13 @@ private static InputStream openStream(String fileName) {
         return result;
     }
 
+    /**
```

Comment:
Actually we want to remove this as we want to use `quarkus.<whatever>` for Quarkus direct properties, and only expose specific alias/mappings to Quarkus properties. So this whole transform key can perhaps just be removed when we start working on the config stuff

Reply:
I agree. I tried to reference this in the comment as a reminder when doing the config stuff.